docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: "2.1"
+
+services:
+  fbcp:
+    image: balenablocks/fbcp
```

Comment:
Do you need a tag here? We need the Docker-compose to function in order to build the block as an app in balenaCloud. The cards on hub are generated from the apps.

Reply:
@phil-d-wilson no we do not. the image without a tag contains all the prebuilt binaries for different target screens. End users have to specify which one to load via the `FBCP_DISPLAY` env var which i've left empty [here](https://github.com/balenablocks/fbcp/blob/prep-for-hub/balena.yml#L15)

Reply:
Ah, OK. This is why I left this to you. 😁